http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/fault/GoAwayTest.java

```diff
@@ -97,6 +97,21 @@ public void goAwayCanCloseAllStreams() throws InterruptedException {
 
         CountDownLatch allRequestsReceived = new CountDownLatch(2);
         Supplier<Http2FrameListener> frameListenerSupplier = () -> new TestFrameListener() {
+            @Override
+            public void onHeadersRead(ChannelHandlerContext ctx, int streamId, Http2Headers headers, int padding, boolean endStream) {
+                serverChannels.add(ctx.channel().id().asShortText());
+
+                Http2Headers outboundHeaders = new DefaultHttp2Headers()
+                    .status("200")
+                    .add("content-type", "text/plain")
+                    .addInt("content-length", 5);
+
+                frameWriter().writeHeaders(ctx, streamId, outboundHeaders, 0, false, ctx.newPromise());
+                ctx.flush();
+
+                allRequestsReceived.countDown();
+            }
```

Comment:
Can we delegate instead of copy+paste?

Reply:
Do you mean keep the used one instead of both?